zsh-syntax-highlighting.zsh

```diff
@@ -67,7 +76,9 @@ _zsh_highlight()
 
   # Remove all highlighting in isearch, so that only the underlining done by zsh itself remains.
   # For details see FAQ entry 'Why does syntax highlighting not work while searching history?'.
-  if [[ $WIDGET == zle-isearch-update ]] && ! (( $+ISEARCHMATCH_ACTIVE )); then
```

Comment:
@m0vie At this point it may be easier to ask pws to release a 5.3.2 with your patch (and other stability fixes) included?